engine/dlib/wscript

```diff
@@ -99,7 +99,12 @@ def shutdown(sht):
     # write a config file for the tests to use
     config = configparser.RawConfigParser()
     hostname = socket.gethostname()
-    local_ip = socket.gethostbyname(hostname)
+    try:
+        local_ip = socket.gethostbyname(hostname)
+    except socket.gaierror as e:
+        print(e)
+        print(f"Hostname was '{hostname}', now trying empty ('') string now")
+        local_ip = socket.gethostbyname("")
```

Comment:
It seems GitHub has changed something with their CI, and this doesn't always work.
So, falling back to "" will yield another ip (0.0.0.0 by the looks of it), but the tests will run which is the main point.

Locally, we want the proper ip since we run tests between devices (Windows <-> PS4 etc)